modules/installation-osp-running-installer.adoc

```diff
@@ -0,0 +1,67 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+
+[id="installation-osp-running-installer_{context}"]
+= Running the {product-title} installer
+
+ifeval::["{context}" == "installing-openstack-installer"]
+:default-osp:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:custom-osp:
+endif::[]
+
+After you prepare your OpenStack deployment for {product-title} installation and download the installer files, run the interactive installer.
+
+By default, the installer:
+
+* Deploys an isolated tenant network
+* Deploys a bootstrap instance
+* Deploys 3 master nodes
+* After the master nodes are deployed, destroys the bootstrap instance
+* Deploys 3 worker nodes
+
+ifdef::default-osp[]
+If you want to customize your installation, see *Installing a cluster on OpenStack with customizations*.
+endif::[]
+
+.Procedure
+
+. From where you downloaded the installer, run the `openshift-install create cluster` command:
```

Comment:
Before actually running the installer, the LB/API floating IP and its records/hostsfile should be available. Currently this section appears before the one documenting this.

If we don't want to rearrange them, I suggest something like this:

> API access: the installer will try to connect to `api.<cluster name>.<domain>` to report progress. Therefore this record must resolve to a floating IP address that will be attached to the cluster before you run the installer.
> For a successful unattended installation, please create a floating IP address with `openstack floating ip create <external network>` and create a `api.<cluster name>.<domain>` DNS record that resolves to it.
> Then during installation either select this floating IP address from the interactive prompt or set it in the `lbFloatingIP` setting in the `install-config.yaml` file.
> See the **Enabling API access with floating IP addresses** (link) section for more details.

(this is probably too verbose, but it's the amount of information you need before you create the cluster)

Reply:
Bumping this up ahead of running the installer.